cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/loadbalancer/GcpForwardingRuleResourceBuilder.java

```diff
@@ -90,14 +89,14 @@ public List<CloudResource> build(GcpContext context, AuthenticatedContext auth,
                 Integer trafficPort = buildableResource.getParameter(TRAFFICPORT, Integer.class);
                 forwardingRule.setPorts(List.of(String.valueOf(trafficPort)));
             }
-            if (scheme.equals(GcpLoadBalancerScheme.INTERNAL)) {
+            if (scheme.equals(GcpLoadBalancerScheme.INTERNAL) || scheme.equals(GcpLoadBalancerScheme.GATEWAY_INTERNAL)) {
```

Comment:
please consider adding some logs here and for `getForwardinRuleSubnet` to make future investigations easier

Reply:
sure